types/monday-sdk-js/index.d.ts

```diff
@@ -200,7 +200,7 @@ interface MondayClientSdk {
 interface MondayServerSdk {
     setToken(token: string): void;
 
-    api(query: string, options?: Partial<{ token: string }>): Promise<any>;
+    api(query: string, options?: Partial<{ token: string, variables?: object} >): Promise<any>;
```

Comment:
nit: `variables` doesn't need to be made optional because the whole object is a `Partial<>`

```suggestion
    api(query: string, options?: Partial<{ token: string, variables: object}>): Promise<any>;
```